plugin.json

```
@@ -1,7 +1,7 @@
 {
     "id": "com.mattermost.aws-sns",
     "name": "AWS SNS",
-    "description": "Send alert notifications from [Amazon AWS CloudWatch](https://aws.amazon.com/cloudwatch/) to Mattermost channels via AWS SNS",
+    "description": "Send alert notifications from Amazon AWS CloudWatch (https://aws.amazon.com/cloudwatch/) to Mattermost channels via AWS SNS.",
```

nit: do we need to link out to CloudWatch at all? Seems like this is something a customer can Google for themselves (along with "AWS SNS" if that's foreign to them).

```suggestion
    "description": "Send alert notifications from Amazon AWS CloudWatch to Mattermost channels via AWS SNS.",
```